tests/topotests/ospf_basic_functionality/test_ospf_asbr_summary_topo1.py

```diff
@@ -575,18 +575,18 @@ def test_ospf_type5_summary_tc48_p0(request):
     }
     dut = "r1"
     result = verify_ospf_rib(tgen, dut, input_dict, expected=False)
-    assert (
-        result is not True
-    ), "Testcase {} : Failed \n Error: " "Routes still present in OSPF RIB {}".format(
-        tc_name, result
+    assert result is not True, (
+        "Testcase {} : Failed\n Expected: Routes should not present in OSPF RIB \n Error: "
```

Comment:
... Routes should not be present ...

The "be" needs to be added to all of these ... :-)

Reply:
Also need to sort the commit blocker ...

```One of your commits does not have a blank line between the summary and body; this will break git log --oneline```

Reply:
Fixed.